examples/eight_schools/eight_schools.py

```diff
@@ -0,0 +1,88 @@
+"""Implement the stan 8 schools example using the recommended non-centred
+parameterization.
+
+The Stan example is slightly modified to avoid improper priors and
+avoid half-Cauchy priors.  Inference is with Edward using both HMC
+and KLQP.
+
+This model has a hierachy and an inferred variance - yet the example is
+very simple - only the Normal distribution is used.
+
+#### References
+https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started
+http://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import tensorflow as tf
+import numpy as np
+from edward.models import Normal, Empirical
+
+
+def main(_):
+    # data
+    J = 8
+    data_y = np.array([28, 8, -3, 7, -1, 1, 18, 12])
+    data_sigma = np.array([15, 10, 16, 11, 9, 11, 10, 18])
+
+    # model definition
+    mu = Normal([0.], [10.])
```

Comment:
Why not `mu = Normal(0., 10.)`?

Reply:
I get a different result with the two calls:

In [4]: Normal(0., 10.)
Out[4]: <ed.RandomVariable 'Normal/' shape=() dtype=float32>

In [5]: Normal([0.], [10.])
Out[5]: <ed.RandomVariable 'Normal_1/' shape=(1,) dtype=float32>

.. am I missing something?

Reply:
Nope. It being a scalar random variable is what I'm suggesting. :)

This more closely aligns with the Stan program, which uses scalars for the group-level parameters `mu` and `logtau`. A vector of length one adds slightly additional complexity.